src/cpu/acl/matmul/acl_matmul.cpp

```diff
@@ -31,24 +31,36 @@ status_t acl_matmul_t::execute_forward(const exec_ctx_t &ctx) const {
     auto wei_base = CTX_IN_MEM(const data_t *, DNNL_ARG_WEIGHTS);
 
     bool is_transA = pd()->amp_.is_transA;
+    bool is_transB = pd()->amp_.is_transB;
     bool use_dst_acc = pd()->amp_.use_dst_acc;
 
     std::lock_guard<std::mutex> _lock {this->mtx};
     auto *acl_resource = ctx.get_resource_mapper()->get<acl_resource_t>(this);
     acl_matmul_obj_t &acl_obj = acl_resource->get_acl_obj();
-
-    // [WA] ACL Matmul produces wrong results in case it is not reconfigured on each inference
```

Comment:
Why this WA was deleted?
It is enabled in fork only

Reply:
restored